components/eam/src/physics/cam/modal_aer_opt.F90

```diff
@@ -149,8 +156,6 @@ subroutine modal_aer_opt_init()
 
    ! set flags to check aerosol settings
    call rad_cnst_get_info(0, nmodes=nmodes)
```

Comment:
I know we are not touching init routine but I think it will be helpful to make the following variables parameter at the module level:
```
   rmmin = 0.01e-6_r8
   rmmax = 25.e-6_r8
   xrmin = log(rmmin)
   xrmax = log(rmmax)
```